src/connectors/huskrecordings.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+Connector.playerSelector = '#webamp';
+
+Connector.trackSelector = '#title';
+
+Connector.artistSelector = '#artist';
+
+Connector.albumSelector = '#album';
+
+Connector.isPlaying = () => Util.hasElementClass(
+	['#metadata'],
```

Comment:
```suggestion
	'#metadata',
```

Could be simplified since we're only passing in a single selector